src/_scss/pages/covid19/_budgetCategoriesTable.scss

```diff
@@ -0,0 +1,70 @@
+.usda-table {
+    margin-top: 0;
+    td {
+        &:first-child {
+            @include span-columns(6 of 12, table);
```

Comment:
oh snap... what is this? Super nice.

Reply:
@MaxwellKendall Yeah, it is pretty nice! It's part of bourbon apparently. https://neat.bourbon.io/docs/1.8.0/